File: attribouter/src/main/java/me/jfenn/attribouter/wedges/link/GitHubLinkWedge.java
package me.jfenn.attribouter.wedges.link;

import androidx.annotation.NonNull;

public class GitHubLinkWedge extends LinkWedge {

    public GitHubLinkWedge(@NonNull String name, int priority) {
        this(name, priority, false);
    }

    public GitHubLinkWedge(@NonNull String item, int priority, boolean isFullUrl) {
        super("github", "@string/title_attribouter_github", isFullUrl ? item : "https://github.com/" + item, "@drawable/ic_attribouter_github", false, priority);
    }

}

File: attribouter/src/main/java/me/jfenn/attribouter/wedges/GitHubLinkWedge.kt
package me.jfenn.attribouter.wedges

class GitHubLinkWedge(
        item: String, priority: Int, isFullUrl: Boolean = false
) : LinkWedge(
        id = "github",
        name = "@string/title_attribouter_github",
        url = if (isFullUrl) item else "https://github.com/$item",
        icon = "@drawable/ic_attribouter_github",
        priority = priority
)
